torch/csrc/generic/Storage.cpp

```diff
@@ -349,7 +350,7 @@ void THPStorage_(initCopyMethods)()
 #endif
   // add CPU <- GPU copies to base type
   #define THCpuStorage_(name) TH_CONCAT_4(TH, Real, Storage_, name)
-  extern THPCopyList THCpuStorage_(copy_functions);
+  THPCopyList THCpuStorage_(copy_functions);
```

Comment:
This doesn't look right. Previously it was meant to insert extra functions into an extern variable declared previously. Now, this will create a local, and that's almost certainly not what you want.